File: nodejs/7 kyu/170.js
// https://www.codewars.com/kata/5680781b6b7c2be860000036/train/javascript

// Passed

function vowelIndices(word) {
  let vowelPositions = [];
  let vowels = ["a", "e", "i", "o", "u", "y"];
  let currIdx = 0;
  for (const letter of word) {
    if (vowels.includes(letter.toLowerCase())) {
      vowelPositions.push(currIdx + 1);
    }

    currIdx++;
  }

  return vowelPositions;
}

const output = vowelIndices("apple");
console.log(output);

File: python/7 kyu/170.py
# https://www.codewars.com/kata/58a6742c14b042a042000038/train/python

# Passed

def s2n(m, n):
    result = 0
    for base in range(m + 1):
        for power in range(n + 1):
            result += base ** power
            
    return result

output = s2n(3,5)
print(output)